stores/src/main/java/dataBase/DataBaseConnect.java

```diff
@@ -0,0 +1,25 @@
+package dataBase;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DataBaseConnect {
+    private final String driver = "org.h2.Driver";
+    private final String url = "jdbc:h2:tcp://localhost/~/test";
+    private final String user = "sa";
+    private final String password = "";
+    private Connection connection;
```

Comment:
do you really need this param here? what is the purpose of global arg if you do not reuse it or so?